allenact/base_abstractions/task.py

```diff
@@ -68,7 +66,7 @@ def get_observations(self, **kwargs) -> Any:
         return self.sensor_suite.get_observations(env=self.env, task=self, **kwargs)
 
     @property
-    @abstractmethod
+    @abc.abstractmethod
```

Comment:
I don't understand why we are so thorough in importing all we use from typing and then we don't import all we use from abc, but I guess those are accepted conventions

Reply:
Yah, I think this is just a convention in AllenAct (since we use types pretty heavily I just find it unpleasant to have to write `typing.` everywhere, especially when types are nested (e.g. `typing.Dict[typing.Tuple, typing.Sequence]`). Really I wouldn't be opposed to always having `from abc import ...`.